ibm_catalog.json

```diff
@@ -172,66 +164,236 @@
+              "key": "existing_resource_group_name"
             },
             {
-              "key": "public_cert_engine_enabled"
+              "key": "key_management_service_encryption_enabled"
             },
             {
-              "key": "public_cert_engine_internet_services_crn"
+              "key": "service_endpoints",
+              "options": [
+                {
+                  "displayname": "public",
+                  "value": "public"
+                },
+                {
+                  "displayname": "private",
+                  "value": "private"
+                }
+              ]
             },
             {
-              "key": "public_cert_engine_dns_provider_config_name"
+              "key": "existing_secrets_manager_kms_key_crn"
             },
             {
-              "key": "public_cert_engine_lets_encrypt_config_ca_name"
+              "key": "skip_kms_iam_authorization_policy"
             },
             {
-              "key": "acme_letsencrypt_private_key"
+              "key": "ibmcloud_kms_api_key"
             },
             {
-              "key": "private_cert_engine_enabled"
+              "key": "kms_endpoint_type",
+              "options": [
+                {
+                  "displayname": "Public",
+                  "value": "public"
+                },
+                {
+                  "displayname": "Private",
+                  "value": "private"
+                }
+              ]
             },
             {
-              "key": "private_cert_engine_config_root_ca_name"
+              "key": "kms_key_ring_name"
             },
             {
-              "key": "private_cert_engine_config_root_ca_common_name"
+              "key": "kms_key_name"
             },
             {
-              "key": "private_cert_engine_config_root_ca_max_ttl"
+              "key": "enable_event_notifications"
             },
             {
-              "key": "private_cert_engine_config_intermediate_ca_name"
+              "key": "event_notifications_email_list"
             },
             {
-              "key": "private_cert_engine_config_template_name"
+              "key": "event_notifications_from_email"
             },
             {
-              "key": "existing_secrets_manager_kms_key_crn"
+              "key": "event_notifications_reply_to_email"
             },
             {
-              "key": "skip_kms_iam_authorization_policy"
+              "key": "existing_event_notifications_instance_crn"
             },
             {
-              "key": "ibmcloud_kms_api_key"
+              "key": "skip_event_notifications_iam_authorization_policy"
             },
             {
-              "key": "kms_endpoint_type",
+              "key":"cbr_rules"
+            }
+          ],
+          "architecture": {
+            "descriptions": "This architecture supports creating and configuring a Secrets Manager instance.",
+            "features": [
```

Comment:
Can you refactor the way the variation features are displayed, the same way I did in https://github.com/terraform-ibm-modules/terraform-ibm-scc/pull/249